src/victory-legend/victory-symbol.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes } from "react";
```

Comment:
I'm moving primitives from `victory-chart` over to `victory-core` and exposing them for users / devs to take advantage of, so you will be able to use https://github.com/FormidableLabs/victory-chart/blob/master/src/components/victory-scatter/point.js rather than writing something new